dev/tools/localization/bin/gen_date_localizations.dart

```diff
@@ -123,28 +133,67 @@ Future<void> main(List<String> rawArgs) async {
   if (writeToFile) {
     final File dateLocalizationsFile = File(path.join('packages', 'flutter_localizations', 'lib', 'src', 'l10n', 'generated_date_localizations.dart'));
     dateLocalizationsFile.writeAsStringSync(buffer.toString());
-    Process.runSync(path.join('bin', 'cache', 'dart-sdk', 'bin', 'dartfmt'), <String>[
-      '-w',
+    final String extension = Platform.isWindows ? '.exe' : '';
```

Comment:
Seems odd that this has to be done "by hand"

Reply:
We wrote package:process to handle this back in the day